CONTRIBUTING.md

```diff
@@ -66,8 +68,9 @@ If you plan to work on a new, bigger feature, then it might be a good idea to
 discuss it with us early, e.g., by creating a new issue ticket.
 
 [lgtm]:     https://lgtm.com/projects/g/dosbox-staging/dosbox-staging/
```

Comment:
Missed an lgtm reference?  This link redirects to a GitHub blog post now.

Reply:
Cheers, will eradicate this one as well 🐛 🔨 😆